corehq/form_processor/tests/utils.py

```diff
@@ -170,6 +172,10 @@ def _delete_all(db, viewname, **view_kwargs):
 )
 
 
+def sql_backend_case(cls):
```

Comment:
true it used to be `sql_backend_test_case` but then nose kept picking it up as an actual test... so i dropped the test and now the name doesn't really make sense. open to ideas of something without test